### ISSUE TITLE
Mark this repository as deprecated 

### ISSUE THREAD

**Issue by vallemar** [EddyVerbruggen/nativescript-plugin-firebase#1887]
there are a lot of people on NS 8 trying to use this plugin because they don't know there is a new one.  It would be nice to mark it as deprecated and report that the new plugin is this: https://github.com/NativeScript/firebase

**Comment by sockRa** [comment]
I agree. At least add a link to the new repository in the description.
